src/clarity/forms/_forms.clarity.scss

```diff
@@ -434,6 +433,23 @@ $clr-custom-checkbox-radio-top: ($clr_baselineRem_1 - $clr-checkbox-radio-height
             display: inline-block;
         }
 
+        input[type="checkbox"]:indeterminate + label::before {
+            background: $clr-form-field-background-color;
+            border: none;
+        }
+        input[type="checkbox"]:indeterminate + label::after {
+            position: absolute;
+            content: '';
+            display: inline-block;
+            height: ($clr-checkbox-radio-height / 4 + 1);
+            width: $clr-checkbox-radio-height / 2;
+            border-left: none;
+            border-bottom: 2px solid white;
+            top: ($clr_baselineRem_1 - $clr-checkbox-radio-height) / 2;
+            left: $clr-checkbox-radio-height / 4;
```

Comment:
These repetitive operations should be in a scoped var. 

`$clr-checkbox-radio-height / 4`

`$clr-checkbox-radio-height / 2`